skillserver/skillserver.go

```diff
@@ -159,8 +159,9 @@ func verifyJSON(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
 		return
 	}
 
-	// Check the app id
-	if !echoReq.VerifyAppID(Applications[r.URL.Path].(EchoApplication).AppID) {
+	if Applications[r.URL.Path] != nil &&
```

Comment:
I wonder if we should add this nil check logic to `VerifyAppID`? It would then be much cleaner:
```go
if !echoReq.VerifyAppID() {
    HTTPError(...)
}
```